internal/resources/metal/vlans/sweeper.go

```diff
@@ -1,4 +1,4 @@
-package vlan
+package vlans
```

Comment:
This (and the directory name) should be `vlan`; we've been using singular resource names for packages thus far.

Reply:
Changed